docs/src/pages/_app.page.tsx

```diff
@@ -29,11 +43,14 @@ function MyApp({ Component, pageProps }) {
     query: { platform = 'react' },
   } = useCustomRouter();
 
+  const isHomepage = pathname === '/' || pathname === '/[platform]';
+
   const filepath = `/${pathname
     .split('/')
     .filter((n) => n && n !== '[platform]')
     .join('/')}`;
   const [colorMode, setColorMode] = React.useState<ColorMode>('system');
+  const [expanded, setExpanded] = React.useState<boolean>(false);
```

Comment:
FYI: no need to pass the generic to `useState` when using a primitive default value, TS will infer it 😄